### PR TITLE
docs:add headlamp & plugins

### PR DESCRIPTION
**Pull request by jingchu000**
docs:add headlamp & plugins & README_CN

fixes https://github.com/headlamp-k8s/headlamp/issues/930

### CODE REVIEW

**Comment by joaquimrocha** [comment]
Thanks @jingchu000 .
@yolossn / @illume , can you check the backend? I think the docs need some updates and the commits need to be split.

**Comment by jingchu000** [comment]
@illume

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Headlamp <img align="right" width=384 src="docs/headlamp_light.svg">
+
+Headlamp 是一个使用简单且可扩展的Kubernetes web UI。
```
Comment (illume) [inline README_CN.md:3]: @lijianzhi01 I wonder if you would mind reviewing this file?

File: backend/cmd/headlamp.go [hunk 2]
```diff
@@ -39,6 +36,10 @@ import (
 	"k8s.io/client-go/tools/portforward"
 	"k8s.io/client-go/transport/spdy"
 
+	"github.com/headlamp-k8s/headlamp/backend/pkg/cache"
```
Comment (illume) [inline backend/cmd/headlamp.go:39]: For my own interest, can you please explain why these import lines were moved?

File: backend/cmd/headlamp.go [hunk 3]
```diff
@@ -75,6 +76,7 @@ const (
 
 const isWindows = runtime.GOOS == "windows"
 
+// PortForward Information forwarded by Port forwarding
```
Comment (illume) [inline backend/cmd/headlamp.go:79]: I'm not sure about this one:
> PortForward Information forwarded by Port forwarding

How about something like this instead?
> PortForward Information for internal use for each port that is being forwarded.

It adds that it is an internal structure of the ports being forwarded.

File: backend/cmd/headlamp.go [hunk 4]
```diff
@@ -136,6 +140,10 @@ func portforwardstore(p PortForward) {
 	portForwards[p.Cluster] = append(portForwards[p.Cluster], p)
 }
 
+// stopOrDeletePortForward stops or deletes a port forward by its cluster and id.
+// It takes three parameters: cluster is the name of the cluster, id is the unique identifier of the port forward,
+// isStopRequest is a boolean value indicating whether to stop or delete the port forward.
+// It returns an error value indicating whether the operation is successful or not.
```
Comment (illume) [inline backend/cmd/headlamp.go:146]: For your consideration... maybe this instead?
> It returns an error if there is one, or nil if there is not.

**Review by illume — CHANGES_REQUESTED** [review]
Thank you for this.

I left a few comments about some issues the linter found, a few other small things, and also asked someone else to review your README_CN.md.

Joaquim suggested that the commits should be split up into separate ones per topic or change. For example, one commit for the README_CN, one for the headlamp.go stopOrDeletePortForward change. However, if you prefer to not split it up into different commits... let me know and when you've finished all of your changes I can assist doing this for you.

File: backend/cmd/headlamp.go [hunk 5]
```diff
@@ -89,6 +91,8 @@ type PortForward struct {
 	Error            string `json:"error"`
 }
 
+// PortForwardPayload Request parameters for Port forwarding
+// RequestURL:/portforward  POST
```
[the remark this answers is not in the document]
Comment (jingchu000) [reply backend/cmd/headlamp.go:94]: OK

File: backend/cmd/headlamp.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (jingchu000) [reply backend/cmd/headlamp.go]: OK

File: backend/cmd/headlamp.go [hunk 6]
```diff
@@ -148,7 +156,6 @@ func stopOrDeletePortForward(cluster string, id string, isStopRequest bool) erro
 					v.closeChan <- struct{}{}
 					clusterPortForwards[index] = v
 				}
-
```
[the remark this answers is not in the document]
Comment (jingchu000) [reply backend/cmd/headlamp.go]: OK

File: backend/cmd/headlamp.go (hunk 2 above)
Comment (jingchu000) [reply backend/cmd/headlamp.go]: Developer tools caused, recovered

File: backend/cmd/headlamp.go (hunk 3 above)
Comment (jingchu000) [reply backend/cmd/headlamp.go]: OK

**Comment by jingchu000** [comment]
Which question is not still there, could you please prompt me?

**Comment by jingchu000** [comment]
If it's more difficult, should I remove the README_CN?

File: backend/cmd/headlamp.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (jingchu000) [reply backend/cmd/headlamp.go]: Okay, it has been updated

**Comment by jingchu000** [comment]
@illume  Is this okay?

**Review by illume — APPROVED** [review]
👍 thanks.

**Comment by illume** [comment]
> If it's more difficult, should I remove the README_CN?

Good idea, thanks. We have some CN speaking collaborators, so if you want to submit a PR for the README_CN someone will get to it in time.

**Pull request merged**